### DISCUSSION TITLE
Navigating between tree and note window

### DISCUSSION THREAD

**Discussion by dukester** [tsujan/FeatherNotes#349] [General]
Other than using my mouse, can I use my keyboard to navigate between the two? I hate jumping back and forth between mouse & keyboard. TIA ..

**Comment by tsujan** [comment]
Escape/Ctrl+Escape, as is mentioned in Help → Hidden Keyboard Shortcuts.

**Comment by dukester** [comment]
On Sat, 27 Jan 2024 03:11:47 -0800
tsujan ***@***.***> wrote:

> Escape/Ctrl+Escape, as is mentioned in Help → Hidden Keyboard
> Shortcuts.

I'm running v0.4.6 from the antiX Debian Buster repo.
No such mention in  Help → Hidden Keyboard Shortcuts

I tried Esc/^-Esc but no joy. Is my version the latest?
--
Duke

**Reply by tsujan** [reply]
> I'm running v0.4.6...

Old versions are considered "dead". The latest version is 1.1.1. See https://github.com/tsujan/FeatherNotes/releases

**Comment by dukester** [comment]
On Sat, 27 Jan 2024 06:23:30 -0800
tsujan ***@***.***> wrote:

> > I'm running v0.4.6...
>
> Old versions are considered "dead". The latest version is 1.1.1.
> See https://github.com/tsujan/FeatherNotes/releases

Right on - thx!
I just DLed the tar file. Tried to build, but too many dependencies issues! Gave up. so I'll be satisfied with the version I have. Thx anyway!

--
Duke
